### ISSUE TITLE
onSnapPhotoCompleted does not get called

### ISSUE THREAD

**Issue by nimisane** [mtellect/CameraDeepAR#52]
snapPhoto() captures  and saves image but onSnapPhotoCompleted is never gets executed

**Comment by smiteshmore123** [comment]
Yes, I am also facing the same issue. It's frustrating. I am trying since long time. Still no solution
